bitnami/redis-cluster/README.md

```diff
@@ -251,8 +251,8 @@ The following table lists the configurable parameters of the Redis<sup>TM</sup>
 | Parameter                                       | Description                                                                                                                                       | Default        |
 |-------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|----------------|
 | `cluster.init`                                  | Enable the creation of a job that initializes the Redis<sup>TM</sup> Cluster                                                                      | `true`         |
-| `cluster.nodes`                                 | Number of nodes in the Redis<sup>TM</sup> cluster                                                                                                 | `6`            |
-| `cluster.replicas`                              | Number of replicas for every master in the cluster                                                                                                | `1`            |
+| `cluster.nodes`                                 | Total Number of nodes in the Redis<sup>TM</sup> cluster including `replicas`. The cluster has to have at least 3 master nodes, and `nodes = numOfMasterNodes + numOfMasterNodes * replicas`. So `nodes = 6` and `replicas = 1` means 3 master and 3 replica nodes.                                                                                        | `6`            |
```

Comment:
The descriptions in this table should be short. Do you think that we can change it to something like the following?

```suggestion
| `cluster.nodes`                                 | Total Number of nodes in the Redis<sup>TM</sup> cluster including `replicas`. See the "Cluster topology" section                                  | `6`            |
```

Then, you can consider adding more information in https://github.com/bitnami/charts/tree/master/bitnami/redis-cluster#cluster-topology